app/renderer/components/preferences/payment/captcha.js

```diff
@@ -0,0 +1,204 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const {StyleSheet, css} = require('aphrodite/no-important')
+
+// Components
+const ImmutableComponent = require('../../immutableComponent')
+
+// Actions
+const appActions = require('../../../../../js/actions/appActions')
+
+// Constants
+const promotionStatuses = require('../../../../common/constants/promotionStatuses')
+
+// Styles
+const cx = require('../../../../../js/lib/classSet')
```

Comment:
Were we deprecating the use of className cx()? It's implied from the styling guidelines but I don't know for sure.

Reply:
not sure

Reply:
removed